parachain/pallets/control/src/lib.rs

```diff
@@ -424,6 +426,32 @@ pub mod pallet {
 
 			Self::do_transfer_native_from_agent(agent_id, para_id, recipient, amount, pays_fee)
 		}
+
+		/// Sends a message to the Gateway contract to update fees
+		///
+		/// Privileged. Can only be called by root.
+		///
+		/// Fee required: No
+		///
+		/// - `origin`: Must be root
+		/// - `location`: Location used to resolve the agent
+		/// - `recipient`: Recipient of funds
+		/// - `amount`: Amount to transfer
+		#[pallet::call_index(8)]
+		#[pallet::weight(T::WeightInfo::set_token_transfer_fees())]
+		pub fn set_token_transfer_fees(
+			origin: OriginFor<T>,
+			register: u128,
+			send: u128,
```

Comment:
Lets validate these to make sure they are non-zero, and if not, return an `InvalidTokenFees` error.

Reply:
https://github.com/Snowfork/snowbridge/pull/991/commits/0a2cb1682245675f5189147702a126583eccc1c2